README.md

```diff
@@ -282,17 +288,51 @@ too (`-lkstat`).
 
 [Disabling CPU Frequency Scaling](#disabling-cpu-frequency-scaling)
 
+<a name="running-benchmarks" />
+
+### Running Benchmarks
```

Comment:
I think it would make sense to keep this after `Output Files`, before `Running a Subset of Benchmarks`